homeassistant/components/atag/.translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "title": "Atag",
+    "step": {
+      "user": {
+        "title": "Connect to the device",
+        "data": {
+          "host": "Host",
+          "email": "Email (optional)",
```

Comment:
```suggestion
```

This isn't used

Reply:
Removed, may need to be added back in when a future version supports remote connectivity (requires the email)